SQL/New_patches/2023-03-14_EEG_Acquisition_instrument.sql

```diff
@@ -0,0 +1,84 @@
+CREATE TABLE `EEG_Acquisition` (
```

Comment:
I don't think this belongs here. It's a sample RB instrument. @ridz1208 how do we usually handle those?

Also it would be a good idea to make it a json data instrument so that no table is required.

Reply:
If it removes the need for a table, sure! How can I do a JSON data instrument?